GoogleDataTransport/GDTLibrary/GDTConsoleLogger.m

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-#import "GDTLibrary/Private/GDTConsoleLogger.h"
+#import "GDTLibrary/Public/GDTConsoleLogger.h"
```

Comment:
Why does it need to be public now?

Reply:
Because it'll be used in libraries that depend on this library (GDTCCT and FIRCD, specifically)